Ontology/Willow/Asset/Equipment/HVAC/Chiller/Chiller.json

```diff
@@ -30,6 +30,14 @@
       },
       "writable": true
     },
+    {
+      "@type": "Component",
+      "displayName": {
+        "en": "Evaporator"
+      },
+      "name": "evaporator",
+      "schema": "dtmi:com:willowinc:Evaporator;1"
+    },
```

Comment:
Lines 97 to 123 describe compressor type. I think compressors should be added as components the way condensers and evaporators have, and we can have specific compressor types as components. What do you think?

Reply:
Yes, I like that for consistency, but are there more properties for these types of compressors that are important for our analytics? This would also introduce a breaking change so would need to see how extensively this property is defined.